drivers/volume/aws/aws.go

```diff
@@ -672,13 +673,17 @@ func (a *aws) IsVirtualMachineSupported() bool {
 }
 
 func init() {
-	a := &aws{}
-	err := a.Init(nil)
-	if err != nil {
-		logrus.Debugf("Error init'ing aws driver: %v", err)
-	}
-	if err := storkvolume.Register(storkvolume.AWSDriverName, a); err != nil {
-		logrus.Panicf("Error registering aws volume driver: %v", err)
+	if os.Getenv("SKIP_AWS_DRIVER_INIT") == "true" {
+		logrus.Infof("Skipping aws driver init")
+	} else {
+		a := &aws{}
+		err := a.Init(nil)
+		if err != nil {
+			logrus.Debugf("Error init'ing aws driver: %v", err)
```

Comment:
Is there any reason to move to registering the driver if we have already failed here? Should we be panicking here if the initialisation fails?

Reply:
I agree it can be an incomplete init and later accessing the driver may fail elsewhere. But for now I don't want to change anything in the existing code flow. We can discuss on this later with the backup team.